src/java-runtime/java/mono/android/MonoPackageManager.java

```diff
@@ -26,9 +26,20 @@ public class MonoPackageManager {
 
 	static android.content.Context Context;
 
-	public static void LoadApplication (Context context, ApplicationInfo runtimePackage, String[] apks)
+	public static void LoadApplication (Context context)
```

Comment:
This API is "internal" and fine to change parameters?

Reply:
It *should* be "internal", in that "nobody other than us" should be invoking it…

If nobody hits it in .NET 10 previews, it's fine! 🤪